test/test-cmd-add.ts

```diff
@@ -262,6 +262,9 @@ describe("cmd-add.ts", function () {
       await mockProject.tryAssertManifest((manifest) =>
         shouldHaveDependency(manifest, packageA, fileUrl)
       );
+      await mockProject.tryAssertManifest((manifest) =>
```

Comment:
Let's add the same assert for tests `"add pkg@http"` and `"add pkg@git"`.

Reply:
Fixed in 80ac9b2